quienesquien/client.py

```
@@ -1,51 +1,170 @@
+            params['list'] = ','.join(search_list)
+
+        headers = {'Authorization': f'Bearer {token}'}
+
+        try:
+            response = await self._make_request(
+                'GET', search_url, params=params, headers=headers
+            )
+        except httpx.HTTPStatusError as exc:
+            if exc.response.status_code == 401:
+                self._invalidate_auth_token()
+                raise InvalidTokenError(exc.response)
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)